radis/spectrum/rescale.py

```diff
@@ -1712,7 +1712,7 @@ def _recalculate(
             slit_unit = spec.conditions["slit_unit"]
             norm_by = spec.conditions["norm_by"]
             try:
-                shape = spec.conditions["shape"]
+                shape = spec.conditions["slit_shape"]
             except KeyError:
                 shape = "triangular"
```

Comment:
Can we also add a UserWarning warning here ? ("Recomputing with `slit_shape` not given. Assuming a triangular slit.")

Reply:
Done